### PR TITLE
Run `cargo clippy` for PR runs

### PR DESCRIPTION
**Pull request by bencord0**

### CODE REVIEW

File: src/core/errors.rs [hunk 1]
```diff
@@ -130,28 +130,29 @@ impl fmt::Display for CliError {
             CliError::Io(ref err) => {
                 let knd = err.kind();
                 if knd == io::ErrorKind::PermissionDenied {
-                    warn!("If you are on norman - ensure you have access to clean ./OUTPUT and \
-                           ./INPUT");
+                    warn!(
+                        "If you are on norman - ensure you have access to clean ./OUTPUT and \
```
Comment (clux) [inline src/core/errors.rs]: lol, this is still the error string!? s/norman/protected filesystem/ maybe.

File: src/core/input.rs [hunk 2]
```diff
@@ -59,7 +59,7 @@ pub struct InputDependency {
     pub missing: bool,
     pub extraneous: bool,
     pub development: bool,
-    pub version: String, // on disk
+    pub version: String,             // on disk
```
Comment (clux) [inline src/core/input.rs:59]: wtf what options do you have for rustfmt? should we align this?

File: src/main.rs [hunk 3]
```diff
@@ -247,292 +270,452 @@ fn main() {
...
+            SubCommand::with_name("list-dependencies")
+                .setting(AppSettings::Hidden)
+                .arg(
+                    Arg::with_name("core")
+                        .short("c")
+                        .long("core")
+                        .help("Only list core dependencies"),
+                )
+                .about("list dependencies from the manifest"),
+        );
```
Comment (clux) [inline src/main.rs]: ugh, i always slap a `#[rustfmt::skip]` on the clap function, but whatever

File: src/upgrade.rs [hunk 4]
```diff
@@ -116,8 +120,8 @@ fn upgrade_exe(latest: &LatestLal, exe: &ExeInfo) -> LalResult<()> {
     http_download_to_path(&latest.url, &tar_dest)?;
     info!("Backing up {} to {}", exe.path, old_file.display());
     fs::rename(&exe.path, &old_file)?; // need to undo this if we fail
-    // NB: DO NOT INSERT CALLS THAT CAN FAIL HERE BEFORE THE OVERWRITE
-    // 3. force dump lal tarball into exe.prefix - rollback if it failed
+                                       // NB: DO NOT INSERT CALLS THAT CAN FAIL HERE BEFORE THE OVERWRITE
+                                       // 3. force dump lal tarball into exe.prefix - rollback if it failed
```
Comment (clux) [inline src/upgrade.rs]: go home rustfmt, you're drunk

**Review by clux — APPROVED** [review]
generally sensible work by rustfmt, left some comments - but merge away

File: src/upgrade.rs (hunk 4 above)
Comment (bencord0) [reply src/upgrade.rs]: I added an extra line here, seems to have made it a bit better.

File: src/core/input.rs (hunk 2 above)
Comment (bencord0) [reply src/core/input.rs]: tempted to "just do what the formatter says" on this.

**Pull request merged**